packaging/linux/org.shotcut.Shotcut.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<mime-info xmlns='http://www.freedesktop.org/standards/shared-mime-info'>
+  <mime-type type="application/vnd.mlt-project">
+    <comment>MLT project</comment>
```

Comment:
I consistently use "MLT XML" as the standard name for these files in Shotcut, Info.plist, and both the Shotcut and MLT websites.